### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stenver**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,23 @@ The process should pop up on that page almost immediately.
 * `-p, --port`: The devtools protocol port to use, default: 8058
 * `-b, --brk`: Pause on the first line of the script
 
+## Examples:
+Using bugger with popular frameworks is easy and it is a lot faster then using node-inspector.
+
+### Jest:
+Run it with `node`:
+```bash
+node --harmony bugger --brk $(which jest) --runInBand
```
Comment (jkrems) [inline README.md]: I think this might be missing a `$(which bugger)`? Didn't try it though.

**Comment by jkrems** [comment]
Thanks, good idea to document it! One small question before I merge. :)

**Comment by stenver** [comment]
You were right. I don't use global packages myself, so I just assumed it works. I also had a look at the jest documentation and noticed they suggest to install jest locally, so I changed the example to reflect that.

I tested both the jest and mocha commands out and they work now.

**Comment by jkrems** [comment]
Thanks!

**Pull request merged**